### ISSUE TITLE
Invoking getFontDefault twice will cause a core dump

### ISSUE THREAD

**Issue by Capital-EX** [Anut-py/h-raylib#13]
`getFontDefault` seems to cause Raylib's text utilities to suddenly break `measureText`. Additionally, I've found that invoking `getFontDefault` twice will crash Haskell citing a double free error or memory corruption as a cause. I'm not to sure what the cause is, but my guess is the original font is somehow getting destroyed as it is passed to Haskell.

Minimum repoduction:
```hs
main :: IO ()
main = do
  _ <- initWindow 1 1 ""
  defaultFont <- getFontDefault
  defaultFont <- getFontDefault
  return ()
```

**Comment by Capital-EX** [comment]
After poking around, I have found `h-raylib` is in fact destroying the default font. Raylib allocates multiple pieces of data for the default font when the library is loaded. This data must be copied or the global `defaultFont` value will be corrupted. I can make a PR with these changes.

**Comment by Anut-py** [comment]
Fixed in `4.6.0.1`
